src/Kernel-Tests.package/DatePrintFormatTester.class/instance/assert..st

```diff
@@ -0,0 +1,5 @@
+asserting
+assert: aBoolean
+	"We check for true explicitly to make the assertion fail for non booleans"
+	aBoolean == true ifFalse: [self signalFailure: 'Assertion failed']
+			
```

Comment:
All these methods are flattened ones, are they? If so, this is ok.

Reply:
yes